tools/build/libwebrtc/config.sh

```diff
@@ -7,7 +7,7 @@
 
 set -o errexit
 set -o nounset
-set -o pipefail
+#set -o pipefail # This breaks checkout on ADO
```

Comment:
If possible explain how it breaks it exactly

Reply:
Will document. This is related to bad handling in bash which makes in pop in some contexts (like ADO) and not others (like your local bash session). Not sure on the intricate details on why this happens though, but SIGTRAP seems to be actually informational mostly.
- Details : http://www.pixelbeat.org/programming/sigpipe_handling.html
- SIGPIPE bash limitation : https://lists.gnu.org/archive/html/bug-bash/2015-02/msg00052.html